testgrid/config.yaml

```diff
@@ -6690,7 +6690,7 @@ dashboards:
   dashboard_tab:
   - name: conformance
     test_group_name: ppc64le-conformance
-    description: conformance test results for ppc64le
+    description: conformance test results for ppc64le ****This is moved under conformance dashboard, WILL BE REMOVED FROM HERE IN THE FUTURE****
```

Comment:
any reason not to just remove it?

Reply:
Many people are referring this link, I don't want to surprise them immediately.. will remove after sometime.

Reply:
You can leave it in both 🤷‍♂️